packages/language-server/src/documentation.ts

```diff
@@ -39,8 +39,24 @@ export function getNodeDocumentation(
       : EMPTY_STRING;
     contents += NL;
   }
-  // TODO add experimental when it's added to the model in this format: Experimental API [since version <since>]
 
+  if (node.experimentalInfo?.isExperimental) {
```

Comment:
Far too many istanbul ignore statements here,,,

Reply:
Well, the lines are actually covered, the ignore is for the branches (since there's no `istanbul ignore if` for ternary expressions). Kind of hard to find the case where there is no since/text for deprecated or experimental.

Reply:
At a minimum this should be documented as it seems like liberally using `istanbul ignore` to avoid writing tests.

But it seems better that this string builder would be tested with a **unit-test** particularly as this is a string that is visible to the end customer in the UI.

Reply:
I agree that we should add tests for this. It's a technical debt.

Reply:
@sapirpol could you please add **unit** tests for this string builder so we won't drag this debt into the future?